auto-configurations/models/spring-ai-autoconfigure-model-vertex-ai/src/main/java/org/springframework/ai/model/vertexai/autoconfigure/gemini/VertexAiGeminiChatProperties.java

```diff
@@ -35,7 +36,10 @@ public class VertexAiGeminiChatProperties {
 
 	/**
 	 * Vertex AI Gemini API generative options.
+	 * 
+	 * @author Hyunsang Han
```

Comment:
The `@author` tag should be used on the class itself instead of the specific method. Could you move this there (line 28)? Thanks!

Reply:
Thank you! 👍